canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -118,6 +148,7 @@ CheckboxsetControl.propTypes = {
 	controller: PropTypes.object.isRequired,
 	controlItem: PropTypes.element,
 	tableControl: PropTypes.bool,
+	disabled: PropTypes.bool,
```

Comment:
Should be removed since this is never used.

Reply:
Removed it.